### ISSUE TITLE
[iOS] mention in adaptive card breaks date function

### ISSUE THREAD

**Issue by matthiasdilger** [MicrosoftDocs/msteams-docs#5480]
Sending the following adaptive card with a mention

```
{
        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
        "type": "AdaptiveCard",
        "version": "1.2",
        "body": [
         {
            "type": "TextBlock",
            "text": "<at>User</a> Your package will arrive on {{DATE(2017-02-14T06:00:00Z, SHORT)}} at {{TIME(2017-02-14T06:00:00Z)}}",
            "wrap": true
        },
          {            
            "type": "TextBlock",
            "text": "**bold**\n\n_italic_",
            "wrap": true             
          }
        ],
        "msteams": {
          "entities": userAboveMentioned,
          "width": "Full"
        }
      }
```

causes Teams iOS to ignore the date function and display the raw text "{{DATE(2017-02-14T06:00:00Z, SHORT)}}"

**Comment by v-chetsh** [comment]
We will check on it from our end and get back to you .

**Comment by matthiasdilger** [comment]
@sayali-MSFT Sorry I realized my ```<at>``` tag was not closed properly in the example

This is the card I send below. The bot mentions my own user, so that my username becomes highlighted in orange.

```
{
        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
        "type": "AdaptiveCard",
        "version": "1.2",
        "body": [
         {
            "type": "TextBlock",
            "text": "<at>Matthias Dilger</at> Your package will arrive on {{DATE(2017-02-14T06:00:00Z, SHORT)}} at {{TIME(2017-02-14T06:00:00Z)}}",
            "wrap": true
        },
          {            
            "type": "TextBlock",
            "text": "**bold**\n\n_italic_",
            "wrap": true             
          }
        ],
        "msteams": {
          "entities": [
            {
              "mentioned":{
                 "id":"29:xxx",
                 "name":"Matthias Dilger"
              },
              "text":"<at>Matthias Dilger</at>",
              "type":"mention"
           }
          ],
          "width": "Full"
        }
      }
```

To reproduce you need to set your own user and id (the part where I wrote xxx), the ```<at>``` tag in the TextBlock needs to match your username.

It looks like this in a channel:
![90415](https://user-images.githubusercontent.com/5625718/158826934-965171af-cc5b-4cbd-8aad-1bff82ae91a0.jpg)

**Comment by sayali-MSFT** [comment]
@matthiasdilger - We are able to repro that issue , We have raised bug for the same and concerned team is looking into it. We will inform you once we get any update.

**Comment by matthiasdilger** [comment]
This issue has been fixed in Teams iOS 4.8.0, thanks
